docs/product-updates/2023-06-07-api-branch-management.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Simplify Your API Workflow with Bump.sh's API New Branch Management Feature
+tags: [New feature]
+image: /files/changelog/temporary-pic.png
+---
+
+[NEED A VISUAL HERE]
+
+We released our latest feature: Branch Management with the [Bump API](https://developers.bump.sh/).
```

Comment:
:tipping_hand_person: suggestion: I would link to the “Branching” endpoints directly

```suggestion
We released our latest feature: Branch Management with the [Bump API](https://developers.bump.sh/group/endpoint-branches).

What do you think?
```